src/main/java/edu/kis/vh/nursery/ListIntegers/IntArrayStack.java

```diff
@@ -0,0 +1,52 @@
+package edu.kis.vh.nursery.ListIntegers;
+
+import edu.kis.vh.nursery.ListInterface;
+
+public class IntArrayStack implements ListInterface {
+
+    public int MAX_AMOUNT_OF_ELEMENTS = 12;
+    final public int[] numbers = new int[MAX_AMOUNT_OF_ELEMENTS];
+
+    private int EMPTY = -1;
```

Comment:
total field was not a constant. Its value was a constant (-1). It would be best if you change it again to 'total' or some better name, e.g., currentIdx and introduce new constant e.g. INITIAL_INDEX or EMPTY_STACK_INDEX.